src/components/bottomSheet/bottom-sheet.js

```diff
@@ -189,16 +197,15 @@ function MdBottomSheetProvider($$interimElementProvider) {
         $animate.enter(backdrop, options.parent, null);
       }
 
-      var bottomSheet = new BottomSheet(element, options.parent);
-      options.bottomSheet = bottomSheet;
+      options.cleanupGestures = options.isLockedOpen ? angular.noop : new BottomSheetGestures(element, options.parent);
```

Comment:
I agree about the naming, but the main reason I decided to default to a noop was to make the API consistent, no matter the options.